lib/LightPanels/LightPanels.js

```diff
@@ -38,63 +35,8 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	defaultKind: LightPanel,
-
-	/**
-	* @private
-	*/
-	handlers: {
-		onSpotlightContainerEnter: 'enter',
-		onSpotlightContainerLeave: 'leave'
-	},
-
-	/**
-	* @private
-	*/
-	animateTo: function (index) {
-		this.updateSpottability(this.index, index);
-		LightPanels.prototype.animateTo.apply(this, arguments);
-	},
-
-	/**
-	* @private
-	*/
-	indexChanged: function (was, is) {
-		this.updateSpottability(was, is);
-		LightPanels.prototype.indexChanged.apply(this, arguments);
-	},
-
-	/**
-	* @private
-	*/
-	updateSpottability: function (from, to) {
-		var panels = this.getPanels(),
-			panelPrev = panels[from],
-			panelNext = panels[to];
-
-		if (panelPrev) {
-			panelPrev.spotlightDisabled = true;
-			if (this._hasSpotlightFocus) {
-				Spotlight.unspot();
-			}
-		}
-		if (panelNext) {
-			panelNext.spotlightDisabled = false;
-		}
-	},
-
-	/**
-	* @private
-	*/
-	enter: function () {
-		this._hasSpotlightFocus = true;
-	},
-
-	/**
-	* @private
-	*/
-	leave: function () {
-		this._hasSpotlightFocus = false;
-	}
+	defaultKind: LightPanel
```

Comment:
This is a good question. I'd normally err on the side of including it, but I'd want to nail down our naming convention first (if I use `moon-lightpanels` it won't be consistent with `enyo-light-panels` or `moon-light-panel`, but those aren't easily changeable right now without some other reworking of the app, not to mention the other Moonstone controls that we should update as well). Might need to take a step back and examine our controls's class names as a whole.